planner/core/logical_plan_builder.go

```diff
@@ -428,6 +429,7 @@ func (ds *DataSource) setPreferredStoreType(hintInfo *tableHintInfo) {
 	}
 	if hintInfo.ifPreferTiFlash(alias) {
 		ds.preferStoreType |= preferTiFlash
+		ds.possibleAccessPaths = append(ds.possibleAccessPaths, &accessPath{isTablePath: true, storeType: kv.TiFlash})
```

Comment:
Do we need to consider if the hints are conflicted? The path has already been added, but the preferStoreType is cleared.